docker/Dockerfile.kinesis

```diff
@@ -0,0 +1,48 @@
+ARG DISTRO=focal
+ARG CORE_BUILDER_IMAGE=abxit/kinesis-core-builder:latest
```

Comment:
TODO: kinesis-core-builder might need to bundle rust

Reply:
as discussed raised ticket https://bullioncapital.atlassian.net/browse/BLC-514 in backlog